tests/Aspire.Playground.Tests/Infrastructure/PasswordGenerator.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography;
+
+namespace SamplesIntegrationTests.Infrastructure;
+
+internal static class PasswordGenerator
```

Comment:
We shouldn't need to copy this. We should be able to reuse what's already in the repo

https://github.com/dotnet/aspire/blob/024b48bf581f6790b0878128c6acff7863d3d5f7/src/Aspire.Hosting/Utils/PasswordGenerator.cs#L11